app/controllers/Contributions.scala

```diff
@@ -106,6 +108,8 @@ class Contributions(paymentServices: PaymentServices, addToken: CSRFAddToken) ex
       .map(mobileRedirectUrl)
       .filter(_ => request.isIos)
 
+    info(s"thank you page displayed for paypal payment id: ${request.id}")
+    putThankYouPage()
```

Comment:
Hi Dom, 
Thanks for this! The cloudwatch metric functions have been renamed "log" instead of put - as per request.